filemanager/process/tests/test_process_upload.py

```diff
@@ -212,6 +212,39 @@
                     ])
 
 
+def compare_file_lists(list1: list, list2: list) -> bool:
+    """
+    Compare two lists of File objects and determine if they are equal.
+
+    Since restore changes the modification time we will ignore this.
+
+    Parameters
+    ----------
+    list1: list
+        List of file objects
+    list2
+        List of file objects
+    Returns
+    -------
+    True if list of File objects is idenical on name, size, checsum fields,
+    otherwise False is returned.
+
+    """
+    names1 = {file.name for file in list1}
```

Comment:
Avoid using the built-in name ``file``. Even this would work:
```suggestion
    names1 = {file_.name for file_ in list1}
```